challenges/objects-arrays.js

```diff
@@ -59,11 +89,21 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+
+const newContact = graduates.forEach(info => {(contactInfo.push(`${info.first_name} ${info.email}`)); 
+});
+
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+
+const newUni = graduates.forEach(info => {
+  (uni.push(`${info.university}`)); 
+});
+
+
 console.log(uni);
```

Comment:
So far you're iterating over the array and just pushing every university to the `uni` array. What's missing is a functionality to check whether that university has a "uni" in it's name. This could be achieved with a conditional logic inside your loop, while utilizing `.includes()` method. Read more about this one https://stackoverflow.com/a/1473742